### ISSUE TITLE
Video window too short - cannot see all of controller

### ISSUE THREAD

**Issue by aptagood** [hoyois/clicktoplugin#112]
The video controller bar that shows the time is partially hidden when viewed in HTML mode with Clicktoplugin 3.0.  I don't recall this being a problem in earlier versions.  Here's what I see with CTP:

![clicktoplugin](https://cloud.githubusercontent.com/assets/11530868/6701369/1096a54a-ccd9-11e4-8201-829f551fcb2a.png)

and this with Flash:
![flash](https://cloud.githubusercontent.com/assets/11530868/6701380/25cc3646-ccd9-11e4-8d02-63f33f95f4fd.png)

[the website is http://www.osho.com/making-love-–-sacred-experience ]

**Comment by hoyois** [comment]
No problem here. What's your Safari version?

**Comment by aptagood** [comment]
Safari Version 6.2.3 (8537.85.12.18)
Thanks!

**Comment by hoyois** [comment]
What's the size of the placeholder before clicking on it?

**Comment by aptagood** [comment]
695x392 (more or less).
It remains the same size after clicking.

**Comment by hoyois** [comment]
But in your screenshot above it's much narrower. At which point does it become narrow?

**Comment by aptagood** [comment]
The placeholder in the first graphic consists of the entire gray rectangle, 695x392, and contains the smaller black rectangle, 489x368 which is the actual video.

**Comment by hoyois** [comment]
The dimensions of the video are inherited from those of the placeholder in CTP's CSS, but maybe there's a conflicting source of CSS. Try it with all other extensions and your user stylesheet disabled.

**Comment by aptagood** [comment]
Ok, I found the culprit: Youtube5 2.6.8 extension.  It had an option turned on to enable it for YouTube videos.  The video on that website was an embedded Youtube video.  Once I turned off that particular option, CTP worked as expected.

Thanks for your help in troubleshooting the issue.  I didn't realize extensions can conflict with each other like this.  Reminds me of the early Mac OS and having to run Conflict Catcher to sort out extension conflicts.
